src/com/softwaret/kpdf/application/Application.kt

```diff
@@ -74,8 +73,8 @@ private fun Application.installFeatures() {
     }
 }
 
-private fun validateCredential(jwtCredential: JWTCredential) =
-    if (User.doesUserExist(jwtCredential.loginFromPayload)) {
+private fun Application.validateCredential(jwtCredential: JWTCredential) =
+    if (instance<UserExistenceService>().doesUserExist(jwtCredential.loginFromPayload)) {
```

Comment:
Wydzielmy te instance<> do vali z getami

Reply:
Also, nie jestem w sumie pewien czy to pyknie. Wołamy `bindDI` już po `installFeatures`

Reply:
Si nie odpaliłem tego bo zasnałem